<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="2" failed="1" total="3" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="LoginSuite" duration-ms="35014" started-at="2019-01-22T01:25:11Z" finished-at="2019-01-22T01:25:46Z">
    <groups>
    </groups>
    <test name="Login" duration-ms="35014" started-at="2019-01-22T01:25:11Z" finished-at="2019-01-22T01:25:46Z">
      <class name="com.qa.meschino.testcases.Login.verifyLogin">
        <test-method status="PASS" signature="run(org.testng.ITestContext)[pri:0, instance:com.qa.meschino.testcases.Login.verifyLogin@192c3f1e]" name="run" is-config="true" duration-ms="16" started-at="2019-01-21T20:25:11Z" finished-at="2019-01-21T20:25:11Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@2fd1731c]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- run -->
        <test-method status="PASS" signature="startup()[pri:0, instance:com.qa.meschino.testcases.Login.verifyLogin@192c3f1e]" name="startup" is-config="true" duration-ms="4" started-at="2019-01-21T20:25:11Z" finished-at="2019-01-21T20:25:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startup -->
        <test-method status="FAIL" signature="LoginTest(java.util.Hashtable)[pri:0, instance:com.qa.meschino.testcases.Login.verifyLogin@192c3f1e]" name="LoginTest" duration-ms="33803" started-at="2019-01-21T20:25:11Z" data-provider="getData" finished-at="2019-01-21T20:25:45Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Password=Test@12, Runmode=Y, Expected=Success, UserName=neil@peter.com, Description=Verify that, system allows to login with valid Credentials, TCID=TC01}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Verify that, system allows to login with valid Credentials]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Verify that, system allows to login with valid Credentials
	at org.testng.Assert.fail(Assert.java:94)
	at com.qa.meschino.testcases.basetest.BaseTest.reportFailure(BaseTest.java:79)
	at com.qa.meschino.testcases.Login.verifyLogin.LoginTest(verifyLogin.java:79)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1194)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginTest -->
        <test-method status="PASS" signature="endTest(org.testng.ITestResult)[pri:0, instance:com.qa.meschino.testcases.Login.verifyLogin@192c3f1e]" name="endTest" is-config="true" duration-ms="1049" started-at="2019-01-21T20:25:45Z" finished-at="2019-01-21T20:25:46Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=LoginTest status=FAILURE method=verifyLogin.LoginTest(java.util.Hashtable)[pri:0, instance:com.qa.meschino.testcases.Login.verifyLogin@192c3f1e] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endTest -->
        <test-method status="SKIP" signature="LoginTest(java.util.Hashtable)[pri:0, instance:com.qa.meschino.testcases.Login.verifyLogin@192c3f1e]" name="LoginTest" duration-ms="4" started-at="2019-01-21T20:25:46Z" data-provider="getData" finished-at="2019-01-21T20:25:46Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Password=Test@123, Runmode=N, Expected=Unsuccessfull, UserName=aneel@x.com, Description=Verify that, system does not allow to login with Invalid Credentials, TCID=TC02}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping Test case  as runmode is No]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping Test case  as runmode is No
	at com.qa.meschino.testcases.Login.verifyLogin.LoginTest(verifyLogin.java:48)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1194)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginTest -->
        <test-method status="PASS" signature="startup()[pri:0, instance:com.qa.meschino.testcases.Login.verifyLogin@192c3f1e]" name="startup" is-config="true" duration-ms="0" started-at="2019-01-21T20:25:46Z" finished-at="2019-01-21T20:25:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startup -->
        <test-method status="PASS" signature="endTest(org.testng.ITestResult)[pri:0, instance:com.qa.meschino.testcases.Login.verifyLogin@192c3f1e]" name="endTest" is-config="true" duration-ms="130" started-at="2019-01-21T20:25:46Z" finished-at="2019-01-21T20:25:46Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=LoginTest status=SKIP method=verifyLogin.LoginTest(java.util.Hashtable)[pri:0, instance:com.qa.meschino.testcases.Login.verifyLogin@192c3f1e] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endTest -->
      </class> <!-- com.qa.meschino.testcases.Login.verifyLogin -->
    </test> <!-- Login -->
  </suite> <!-- LoginSuite -->
  <suite name="AsthmaTrackerSuite" duration-ms="12" started-at="2019-01-22T01:25:46Z" finished-at="2019-01-22T01:25:46Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="12" started-at="2019-01-22T01:25:46Z" finished-at="2019-01-22T01:25:46Z">
      <class name="com.qa.meschino.testcases.AsthmaTracker.test1">
        <test-method status="SKIP" signature="run(org.testng.ITestContext)[pri:0, instance:com.qa.meschino.testcases.AsthmaTracker.test1@672872e1]" name="run" is-config="true" duration-ms="0" started-at="2019-01-21T20:25:46Z" finished-at="2019-01-21T20:25:46Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@791cbf87]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[run mode of the suite is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: run mode of the suite is NO
	at com.qa.meschino.testcases.basetest.BaseTest.run(BaseTest.java:139)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:277)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1194)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- run -->
        <test-method status="SKIP" signature="startup()[pri:0, instance:com.qa.meschino.testcases.AsthmaTracker.test1@672872e1]" name="startup" is-config="true" duration-ms="0" started-at="2019-01-21T20:25:46Z" finished-at="2019-01-21T20:25:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startup -->
        <test-method status="SKIP" signature="t1()[pri:0, instance:com.qa.meschino.testcases.AsthmaTracker.test1@672872e1]" name="t1" duration-ms="2" started-at="2019-01-21T20:25:46Z" finished-at="2019-01-21T20:25:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- t1 -->
        <test-method status="SKIP" signature="quit()[pri:0, instance:com.qa.meschino.testcases.AsthmaTracker.test1@672872e1]" name="quit" is-config="true" duration-ms="0" started-at="2019-01-21T20:25:46Z" finished-at="2019-01-21T20:25:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
        <test-method status="SKIP" signature="endTest(org.testng.ITestResult)[pri:0, instance:com.qa.meschino.testcases.AsthmaTracker.test1@672872e1]" name="endTest" is-config="true" duration-ms="0" started-at="2019-01-21T20:25:46Z" finished-at="2019-01-21T20:25:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endTest -->
      </class> <!-- com.qa.meschino.testcases.AsthmaTracker.test1 -->
    </test> <!-- Test -->
  </suite> <!-- AsthmaTrackerSuite -->
  <suite name="TestNg datadriver Suite" duration-ms="0" started-at="2019-01-22T01:25:47Z" finished-at="2019-01-22T01:25:47Z">
    <groups>
    </groups>
  </suite> <!-- TestNg datadriver Suite -->
</testng-results>
